### DISCUSSION TITLE
registerUrlHandler - Install prerelease extension version

### DISCUSSION THREAD

**Discussion by yuqizhou77** [microsoft/vscode-discussions#276] [Extension Development QnA] [answered]
Hi,

We are working on a url hanlder but it will be in the pre-release version only for now. If the user has not installed the extension before but visit the url, is there any way that we could directly help user install the pre-release version? Thanks!

**Comment by isidorn** [comment, marked as answer]
I think this is not possible. But @sandy081 would know better.

**Reply by sandy081** [reply]
VS Code cannot do it. You can also register an URL handler in release version and then ask user to install the pre-release version.

**Reply by yuqizhou77** [reply]
Thanks! Yeah, this is what are currently doing by asking user to install pre-release in the release version.
